adobe_vipm/adobe/client.py

```diff
@@ -326,6 +326,7 @@ def create_preview_order(
         reseller_country: str,
         customer_id: str,
         order: dict,
```

Comment:
WDYT about to have here also an order_id? Now the only property that is required to use by this function is an order_id :-)